api/config.ts

```diff
@@ -0,0 +1,13 @@
+import { Client, Databases, Account, ID, Query, Storage } from "appwrite";
+
+const url = "https://cloud.appwrite.io/v1";
+
+const project = "66246c40e329b9509f71";
+
+const client: Client = new Client().setEndpoint(String(url)).setProject(String(project));
```

Comment:
I don't think `: Client` or any of the `String()`s are necessary?

Reply:
No, they weren't. Thanks for catching it, Mike, and I was able to remove them.